modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/ChapterIndices.kt

```diff
@@ -0,0 +1,12 @@
+package au.com.shiftyjelly.pocketcasts.models.entity
+
+data class ChapterIndices(private val delegate: List<Int> = emptyList()) : List<Int> by delegate {
+    companion object {
+        fun fromString(value: String?): ChapterIndices {
+            val list = value?.split(",")?.map { it.toInt() } ?: emptyList()
```

Comment:
We could consider mapping `toIntOrNull()` for safety.

```suggestion
            val list = value?.split(",")?.mapNotNull { it.toIntOrNull() } ?: emptyList()
```